vnext/Microsoft.ReactNative.Cxx/ReactContext.h

```diff
@@ -5,6 +5,7 @@
 #ifndef MICROSOFT_REACTNATIVE_REACTCONTEXT
 #define MICROSOFT_REACTNATIVE_REACTCONTEXT
 
+#include <CppWinRTIncludes.h>
```

Comment:
We must not have this file depend on the external to the shared project header files. The plan is to have this shared project distributed with DLLs. Having this dependency makes it impossible to do until we fix it. It is better not to introduce the 'engineering debt' and do it right from the beginning.
One possible solution is to add this file to the MS.RN.Cxx shared project.
The MS.RN.dll includes it anyway for the TurboModule support.

Another concern is that the seemingly innocent ReactContext.h now brings almost the whole XAML framework with it. Is it possible to avoid making it so heavy?